src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/alert-conditions.mdx

```diff
@@ -0,0 +1,394 @@
+
+ When an incident closes automatically:
+
+ 1. The closing timestamp is backdated to the start of the recovery period.
+ 2. The evaluation resets and restarts from when the previous incident ended.
+
+ All conditions have an incident time limit setting that automatically force-close a long-lasting incident.
+
+ New Relic automatically defaults to 3 days and recommends that you use our default settings for your first alert.
+
+ Another way to close an open incident when the signal does not return data is by configuring a `loss of signal` threshold. Refer to the lost signal threshold section above for more details.
+ </Collapser>
+
+
+ <Collapser
+   id="custom-incident-description"
+   title="Send a custom incident description"
+ >
+
+  Since you're creating an alert condition that lets you know if there are any latency issues with your `WebPortal` application, you want to make sure your developers have all the information they need when notified about this incident. You will use workflows to notify a team Slack channel when an incident is created.
+
+ Learn more about custom incident descriptions in our [docs](/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/advanced-techniques/alert-custom-incident-descriptions).
+
+</Collapser>
+
+
+ <Collapser
+   id="runbook"
+   title="Add runbook URL"
+ >
+
+If you'd like to link to a runbook, you can put the URL in the runbook URL field.
+
+ </Collapser>
+
+</CollapserGroup>
+</Step>
+</Steps>
+
+## Edit or improve an existing alert condition [#edit-existing-alert-condition]
+
+If you want to edit an alert condition you've already created, you can:
+
+1. Go to [one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > **Alerts & AI**.
+2. Select **Alert Conditions** in the left navigation.
+3. Click on the alert condition you want to edit.
+
+From there, you will see the **Alert conditions details** page. This page contains all the elements you set when you created your condition. You can edit specific aspects of the alert condition by clicking the pencil in the top right of each section.
+
+
+### Signal history [#signal-coverage]
+
+Under **Signal history**, you can see the most recent results for the NRQL query you used to create your alert condition. For this example, you would see the average `pageviews` on the `WebPortal` app for the specific time frame you've set.
+
+For all alert conditions built with NRQL queries, the **Signal history** will be presented with a line chart.
+
+Any alert condition built with a synthetic monitor will be a bit different. This is because synthetic monitors allow you to ping your application from multiple locations, which can produce positive or negative results each time the monitor runs. This data can only be presented with a table.
+
+## Troubleshoot
```

Comment:
Don't forget anchor tags! i also think this could use a noun 'Troubleshoot conditions' or stg